pkg/pillar/zedcloud/deferred.go

```diff
@@ -93,23 +93,24 @@ func CreateDeferredCtx(zedcloudCtx *ZedCloudContext, sentHandler *SentHandlerFun
 func (ctx *DeferredContext) HandleDeferred(event time.Time,
```

Comment:
I think it would make sense to update the comment where it says "try to send all" since it only tries to send the deferred items which were present when the function started and needs to be called again to send any items which were added while it was running.